lib/travis/build/script/node_js.rb

```diff
@@ -169,6 +194,12 @@ def setup_npm_cache
           def iojs_3_plus?
             (config[:node_js] || '').to_s.split('.')[0].to_i >= 3
           end
+
+          def install_yarn
+            sh.cmd    "curl -o- -L https://yarnpkg.com/install.sh | bash", echo: true
```

Comment:
How common is such a need? If possible, I would like us to Do The Reasonable Thing™ automatically. If a different version of yarn is desired, I think the users can do that at their own will. (We do not support different versions of Bundler or Pip, for example.)